Gopkg.toml

```diff
@@ -35,3 +35,7 @@ required = [
 name = "gopkg.in/fsnotify.v1"
 source = "https://github.com/fsnotify/fsnotify.git"
 version="v1.4.7"
+
+[[constraint]]
+  branch = "master"
```

Comment:
This should probably use a version, could we make sure that this dependency it's not already vendored via controller-runtime? I'd prefer to not add new constraint if they are already implicit, especially if they are tied to a k8s version.

Reply:
We've removed it as a `constraint` in favor of implicit. When we ran `dep ensure`, it needed to add `k8s.io/component-base` as a new vendor subtree, so we don't believe it was already referenced by `controller-runtime`.